docs-src/content/getting-started/cubic_docker.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Installing the cubicsvr Image
+weight: 5
+---
+
+The SDK is used to call an instance of cubicsvr using gRPC.  Cobalt distributes a docker image that contains the cubicsvr binary and model files.
+
+<!--more-->
+
+1. Contact Cobalt to get a link to the image file in AWS S3.  This link will expire in two weeks, so be sure to download the file to your own server.
+
+2. Download with the AWS CLI if you have it, or with curl:
+
+    ```bash
+    URL="the url sent by Cobalt"
+    IMAGE_NAME="name you want to give the file (should end with the same extension, usually bz2)"
+    curl $URL -L -o $IMAGE_NAME
+    ```
+   
+2. Load the docker image: 
+
+    ```bash
+    docker load < $IMAGE_NAME
+    ```
+
+    This will output the name of the image (e.g. cubicsvr-demo-en_us-16).
+
+3. Start the cubic service listening:
+
+    ```bash
+    docker run -p 2727:2727 -p 8080:8080 --name cobalt cubicsvr-demo-en_us-16
+    ```
+
+    That will start listening for grpc commands on port 2727 and http requests on 8080, and will stream the debug log to stdout.  (You can replace `--name cobalt` with whatever name you want.  That just provides a way to refer back to the currently running container.)
+
+4. Verify the service is running by calling 
+
+    ```bash
+    curl http://localhost:8080/api/version
+    ```
+
+5. If you want to explore the package to see the model files etc, call the following to open the bash terminal on the previously run image.  Model files are located in the /model directory.
+   
+    ```bash
+    docker exec -it cobalt bash`
+    ```
+   
+## Contents of the docker image
+### Base docker image  
+debian-stretch-slim
+
+### Additional dependency 
+(installed with yum install on centos or apt-get on ubuntu): libgfortran3
```

Comment:
would this look better as a bullet point and with the bit in parenthesis at the end ? 

- libgfortran (installed with yum install on centos or apt-get on ubuntu)

Reply:
I wonder if these two sections look a little empty-ish?
![image](https://user-images.githubusercontent.com/26173976/57757551-83c21a80-7717-11e9-90f0-92b86998acfe.png)

Reply:
Maybe have them as bullet points without any specific headings and reduce the other heading sizes by one ?

```markdown
### Contents of the docker image
- **Base docker image** :  debian-stretch-slim
- **Additional dependency** : libgfortran3 (installed with yum install on centos or apt-get on ubuntu)

#### Cobalt-specific files
```
![image](https://user-images.githubusercontent.com/26173976/57757848-31cdc480-7718-11e9-8c25-ea09aee411d7.png)